conda/cli/main.py

```diff
@@ -63,15 +63,17 @@ def generate_parser(*args, **kwargs):
 def main_subshell(*args, post_parse_hook=None, **kwargs):
     """Entrypoint for the "subshell" invocation of CLI interface. E.g. `conda create`."""
     # defer import here so it doesn't hit the 'conda shell.*' subcommands paths
+    from ..base.context import context
     from .conda_argparse import generate_parser
 
-    args = args or ["--help"]
+    # This needs to be called to give the argument parser access to context variables
+    context.__init__()
```

Comment:
This is weird 😭. Any suggestions of how to change it? I don't think the performance penalty is that big, so it might not be that bad to leave in.

Reply:
Please don't do that, that's the fundamental problem of the CLI and context system, we can't change the stack order of the CLI/config system, for risk of triggering side-effects that are related to config loading.

Let's skip the `disabled_plugins` config option in favor of CLI options `--no-plugins` and maybe `--disable-plugin=my-example-plugin --disable-plugin=conda-libmamba-solver`